pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please remove this entire pom and flatten the project to have only the mysql-dsb as root pom. this pom adds nothing.. lets talk

Reply:
Thanks for fixing

Reply:
At your service